File: 01 - Iniciante/1133 - Resto da Divisao/problem1133.c
#include <stdio.h>

int main() {

    int num1, num2, maior, menor;
    scanf("%d", &num1);
    scanf("%d", &num2);

    maior = num1 > num2 ? num1 : num2;
    menor = num1 < num2 ? num1 : num2;

    for (int num = menor+1; num < maior; num++) {
        if (num % 5 == 2 || num % 5 == 3) {
            printf("%d\n", num);
        }
    }
}

File: 01 - Iniciante/1133 - Resto da Divisao/problem1133.py

num1 = int(input())
num2 = int(input())

maior = num1 if num1 > num2 else num2
menor = num1 if num1 < num2 else num2

for num in range(menor+1, maior):
    if num % 5 == 2 or num % 5 == 3:
        print(num)